db/seeds/posts/global_faq.html

```diff
@@ -10,7 +10,7 @@ <h2 id="what-are-trust-levels-and-how-are-they-calculated-">What are trust level
 <!-- I'm omitting any discussion of reputation for now since consensus seems to be that it is beyond MVP. If that is implemented we can add information here. -->
 <p>Trust levels are the system Codidact uses to determine user privileges. Everyone starts at Trust Level 0 and gradually gains the ability to contribute at a higher level. Trust levels are earned by positively contributing to the site. For more information, see <a href="https://github.com/codidact/docs/wiki/User-Privileges">https://github.com/codidact/docs/wiki/User-Privileges</a>. </p>
 <h2 id="what-does-it-mean-that-codidact-is-community-controlled-what-makes-it-different-from-any-other-q-a-site-">What does it mean that Codidact is community-controlled? What makes it different from any other Q&amp;A site?</h2>
-<p>Most Q&amp;A sites are run by companies whose primary goal is to make a profit for their shareholders. This leads to a situation where bringing in revenue can be more important than the health of the community using the site. On Codidact, the users control the sites they use, so there&#39;s no conflict of interest between quality and the bottom line.</p>
+<p>Most Q&amp;A communities are run by companies whose primary goal is to make a profit for their shareholders. This leads to a situation where bringing in revenue can be more important than the health of the community using the site. On Codidact, the users control the communities they use, so there&#39;s no conflict of interest between quality and the bottom line.</p>
```

Comment:
I would leave the *first* one as "sites" -- Q&A run by companies like Quora or SE aren't about the *communities*; they're just Q&A *sites*.  Codidact, on the other hand, is community-centric, so contrasting other sites with our communities is good.  (In other words, leave the second change in this line as communities; this comment is only about the first one.)